src/services/infrastructure/url-generator/url.generator.service.ts

```
@@ -152,91 +77,32 @@ export class UrlGeneratorService {
-          this.FIELD_PROFILE_ID,
           profile.id
         );
-        return this.createUrlFoUserNameID(userEntityInfo.entityNameID);
+        return this.createUrlForUserNameID(userEntityInfo.entityNameID);
       case ProfileType.VIRTUAL_CONTRIBUTOR:
-        const vcEntityInfo = await this.getNameableEntityInfoOrFail(
+        const vcEntityInfo = await this.getNameableEntityInfoForProfileOrFail(
           'virtual_contributor',
-          this.FIELD_PROFILE_ID,
           profile.id
         );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)